sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -111,7 +111,7 @@ case class CreateViewCommand(
 
     // When creating a permanent view, not allowed to reference temporary objects.
     // This should be called after `qe.assertAnalyzed()` (i.e., `child` can be resolved)
-    verifyTemporaryObjectsNotExists(catalog, isTemporary, name, child)
+    verifyTemporaryObjectsNotExists(catalog, isTemporary, name, analyzedPlan)
```

Comment:
can we remove `L98` to `L101`? It seems we don't need to analyze the plan inside `run` method again.